.github/workflows/ci.yaml

```diff
@@ -234,20 +236,22 @@ jobs:
             ./target/${{ matrix.target }}/release/clarinet test --manifest-path examples/${testdir}/Clarinet.toml
           done
 
-      # - name: Functional Tests (Windows)
-      #   if: matrix.os == 'windows-latest'
-      #   run: |
-      #     foreach($testdir in Get-ChildItem examples) {
-      #       ./target/${{ matrix.target }}/release/clarinet test --manifest-path examples/${testdir}/Clarinet.toml
-      #     }
+      - name: Functional Tests (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          foreach($testdir in Get-ChildItem examples) {
+            ls
+            ls $testdir
+            ./target/${{ matrix.target }}/release/clarinet test --manifest-path examples/${testdir}/Clarinet.toml
+          }
 
       - name: Upload Artifacts to GH release
         uses: svenstaro/upload-release-action@v2
         if: startsWith(github.ref, 'refs/tags/v')
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: ${{ matrix.platform }}.zip
-          asset_name: ${{ matrix.platform }}
+          asset_name: clarity-${{ matrix.platform }}.zip
```

Comment:
clarinet :)

Reply:
haha I've made this typo in more places than I'd like to admit. Muscle memory!